conda_build/metadata.py

```diff
@@ -276,7 +276,13 @@ def check_fields(meta, strictness=.9):
         _ = _meta['package']['name']
         _ = _meta['package']['version']
     except KeyError:
-        sys.exit("Invalid meta.yaml: Missing package/name or package/version.")
+        sys.exit('Invalid meta.yaml: Missing package/name or package/version.')
+
+    try:
+        _ = _meta['about']['home']
+        _ = _meta['about']['license_family']
```

Comment:
Should we check for `license_family` or just `license`? Personally, I would vote for the latter.